### PR TITLE
8270336: [TESTBUG] Fix initialization in NonbranchyTree

### PR DESCRIPTION
**Pull request by lmesnik**
The nsk.share.gc.NonbranchyTree might call initTree() twice and incorrectly reports branchiness in the case of IllegalArgumentException(...).

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8270336](https://bugs.openjdk.java.net/browse/JDK-8270336): [TESTBUG] Fix initialization in NonbranchyTree

### Reviewers
 * [Kim Barrett](https://openjdk.java.net/census#kbarrett) (@kimbarrett - **Reviewer**) ⚠️ Review applies to 4782c3f1720f4c7c12d7f4b4513d1f761c65e01a
 * [Thomas Schatzl](https://openjdk.java.net/census#tschatzl) (@tschatzl - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk pull/4761/head:pull/4761` \
`$ git checkout pull/4761`

Update a local copy of the PR: \
`$ git checkout pull/4761` \
`$ git pull https://git.openjdk.java.net/jdk pull/4761/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 4761`

View PR using the GUI difftool: \
`$ git pr show -t 4761`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk/pull/4761.diff">https://git.openjdk.java.net/jdk/pull/4761.diff</a>

</details>

### CODE REVIEW

File: test/hotspot/jtreg/vmTestbase/nsk/share/gc/NonbranchyTree.java [hunk 1]
```diff
@@ -62,36 +59,26 @@
...
-                                             + "least 1.");
+                    + numberOfNodes + ", must be at least 1.");
         }
-        if ( (branchiness >= 1) || (branchiness <= 0) ) {
+        if ((branchiness >= 1) || (branchiness <= 0)) {
             throw new IllegalArgumentException("Illegal value of branchiness: "
-                                             + numberOfNodes + ", must be at "
-                                             + "greater than 0 and less than "
-                                             + " 1.");
+                    + branchiness + ", must be at greater than 0 and less than 1.");
```
Comment (kimbarrett) [inline test/hotspot/jtreg/vmTestbase/nsk/share/gc/NonbranchyTree.java:71]: s/be at/b/

**Review by kimbarrett — APPROVED** [review]
One trivial grammar fix for a pre-existing problem in an error message.  Otherwise looks good.

**Review by tschatzl — APPROVED** [review]
Looks good.

There is some typo in other code: s/Initizlize/Initialize. I do not need a re-review for that fix.

**Pull request closed without merging**